verify-sna/functions/verify-start.js

```diff
@@ -26,46 +26,47 @@
  * }
  */
 
-const fs = require('fs');
-const path = require('path');
-const os = require('os');
-const sqlite3 = require('sqlite3');
-
 const assets = Runtime.getAssets();
-const {
-  connectToDatabaseAndRunQueries,
-  verificationStartDatabaseUpdate,
-} = require(assets['/helpers/db.js'].path);
+const { createVerification } = require(assets['/services/verifications.js']
+  .path);
+const { detectMissingParams } = require(assets['/services/helpers.js'].path);
 
 // eslint-disable-next-line consistent-return
 exports.handler = async function (context, event, callback) {
   const response = new Twilio.Response();
   response.appendHeader('Content-Type', 'application/json');
 
+  const missingParams = detectMissingParams(
+    ['countryCode', 'phoneNumber'],
+    event
+  );
+  if (missingParams.length > 0) {

```

Comment:
This logic is also repeated in the `verify-check.js` file, any chance to move it to a helper method and reuse it?